hadoop-ozone/dist/src/main/smoketest/admincli/replicationmanager.robot

```diff
@@ -30,16 +30,16 @@ Check replicationmanager with explicit host
                         Should contain   ${output}   ReplicationManager
                         Should contain   ${output}   Running
 
-Start replicationmanager
-    ${output} =         Execute          ozone admin replicationmanager start
-                        Should contain   ${output}   Starting ReplicationManager
-                        Wait Until Keyword Succeeds    30sec    5sec    Execute          ozone admin replicationmanager status | grep -q 'is Running'
-
 Stop replicationmanager
     ${output} =         Execute          ozone admin replicationmanager stop
                         Should contain   ${output}   Stopping ReplicationManager
                         Wait Until Keyword Succeeds    30sec    5sec    Execute          ozone admin replicationmanager status | grep -q 'is Not Running'
 
+Start replicationmanager
```

Comment:
hmm i'm curious.. before this change, the replicationmanager was stopped when admincli robot test finish? and this wasn't failing other tests after this?

Reply:
Thanks @smengcl for the review.

> before this change, the replicationmanager was stopped when admincli robot test finish?

Yes.

> and this wasn't failing other tests after this?

I don't think any acceptance test triggered replication so far.

Reply:
Got it. Thanks @adoroszlai !